### DISCUSSION TITLE
Duplicate scoreboard ranks

### DISCUSSION THREAD

**Discussion by osnesone** [ppy/osu#20886] [Q&A] [answered]
[![osuduplicatescoreboardbug3.png](https://i.postimg.cc/5yYMLmrY/osuduplicatescoreboardbug3.png)](https://postimg.cc/q6TYd244)
Hello, I already posted [here](https://osu.ppy.sh/community/forums/topics/1664418?n=1), please read that one first.
The [screenshot](https://postimg.cc/q6TYd244) of it happening on [this](https://osu.ppy.sh/beatmapsets/1017565#osu/2131400) beatmap.
If further information is needed please let me know and I will do my best to add it.

Logs from the new beatmap.
[database.log](https://github.com/ppy/osu/files/9846465/database.log)
[input.log](https://github.com/ppy/osu/files/9846467/input.log)
[runtime.log](https://github.com/ppy/osu/files/9846464/runtime.log)
[network.log](https://github.com/ppy/osu/files/9846461/network.log)
[performance.log](https://github.com/ppy/osu/files/9846466/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/9846468/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/9846469/performance-update.log)
[performance-audio.log](https://github.com/ppy/osu/files/9846471/performance-audio.log)
[performance-input.log](https://github.com/ppy/osu/files/9846472/performance-input.log)

**Comment by peppy** [comment, marked as answer]
Nothing here is loading but I replied in your forum thread. This looks like a misunderstanding rather than an issue.
